api/kessel/inventory/v1beta2/reporter_reference.pb_test.go

```diff
@@ -8,49 +8,49 @@ import (
 )
 
 func TestReporterReference_BasicFields(t *testing.T) {
-	instID := "hbi"
+	instID := "3c4e2382-26c1-11f0-8e5c-ce0194e9e144"
```

Comment:
Can you also add tests with the "redhat.com" scenario? To test that we are allowing dots.